backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/aggregation/MergeStep.java

```diff
@@ -16,6 +17,7 @@ enum MergeStep implements DateAggregationStep {
 	NODE_NO_OVERLAP("_node_no_overlap", NodeNoOverlapCte::new, INTERMEDIATE_TABLE),
 	MERGE("_merge", MergeCte::new, OVERLAP);
 
+	private static final Set<MergeStep> REQUIRED_STEPS = Arrays.stream(values()).collect(Collectors.toSet());
```

Comment:
Ich frage mich gerade ob Set.of nicht das gleiche bewirkt?

Reply:
Ja du hast Recht, danke dir 👍